WordPress/src/main/java/org/wordpress/android/ui/people/PeopleAdapter.java

```diff
@@ -0,0 +1,97 @@
+package org.wordpress.android.ui.people;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.TextView;
+
+import org.wordpress.android.R;
+import org.wordpress.android.models.Person;
+import org.wordpress.android.util.GravatarUtils;
+import org.wordpress.android.util.StringUtils;
+import org.wordpress.android.widgets.WPNetworkImageView;
+
+import java.util.List;
+
+public class PeopleAdapter extends BaseAdapter {
+    private final LayoutInflater mInflater;
+    private List<Person> mPeopleList;
+    private int mAvatarSz;
+
+    public PeopleAdapter(Context context, List<Person> peopleList) {
+        mAvatarSz = context.getResources().getDimensionPixelSize(R.dimen.avatar_sz_medium);
+        mInflater = LayoutInflater.from(context);
+        mPeopleList = peopleList;
+    }
+
+    public void setPeopleList(List<Person> peopleList) {
+        mPeopleList = peopleList;
+        notifyDataSetChanged();
+    }
+
+    @Override
+    public int getCount() {
+        if (mPeopleList == null) {
+            return 0;
+        }
+        return mPeopleList.size();
+    }
+
+    @Override
+    public Person getItem(int position) {
+        if (mPeopleList == null) {
+            return null;
+        }
+        return mPeopleList.get(position);
+    }
+
+    @Override
+    public long getItemId(int position) {
+        Person person = getItem(position);
+        if (person == null) {
+            return 0;
```

Comment:
Is `0` ever a valid person ID? If so, it might be better to return `-1` when person is null.